pkg/network/protocols/http/ebpf_javatls.go

```diff
@@ -78,7 +103,37 @@ func (p *JavaTLSProgram) GetAllUndefinedProbes() (probeList []manager.ProbeIdent
 	return
 }
 
+func allowBlockAttachment(pid uint32) (reject bool) {
```

Comment:
change to the below, and add documentation
```suggestion
func allowBlockAttachment(pid uint32) bool {
```

Reply:
I don't think there a value to remove a named variable give you documentation without reading the whole comment of a function
and in one line you can understand the purpose and what the return mean, specially for bool type
no linter reject this style

Reply:
IMHO, it fits the scenario where you have a very simple function, but that's not the case here
without the documentation how could we understand that `allow` regex has higher priority than the `block` regex?

and regardless, I think that the bool returned from `allowBlockAttachment` is sufficient to understand that true means "reject", so there is no special context being added here
it can fit simple functions with multiple or "not obvious" return values